CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
```

Comment:
sure, that certainly works! I can make the check flexible to detect either the CHANGELOG.md or look for the `semantic-release` run command in package.json (unless there is a better way to detect how a project follows semantic release)

_p.s. looking for the run command is how i detect if the package has a linter and test runner setup as well, i thought that was more flexible than looking for devDeps in package.json_

Reply:
what does the check look for today, just for the existence of the file?
I wouldn't bother checking for the semantic-release in package.json, as it can be very dynamic configuration on package.json and will just set you up for many edge cases to support.